File: hackerrank/30 days of code/day 3, intro to conditional statements/intro_to_conditional_statements.go
// https://www.hackerrank.com/challenges/30-conditional-statements/problem?isFullScreen=true

package main

import "fmt"

func main() {
	var n int
	fmt.Scan(&n)
	isWeird := n&1 == 1 || n >= 6 && n <= 20
	if isWeird {
		fmt.Println("Weird")
	} else {
		fmt.Println("Not Weird")
	}
}

File: hackerrank/30 days of code/day 3, intro to conditional statements/intro_to_conditional_statements.cs
// https://www.hackerrank.com/challenges/30-conditional-statements/problem?isFullScreen=true

using static System.Console;

static class Solution
{
    static void Main()
    {
        int n = int.Parse(ReadLine());
        bool isWeird = n % 2 == 1 || n >= 6 && n <= 20;
        WriteLine(isWeird ? "Weird" : "Not Weird");
    }
}
